app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionChart.jsx

```diff
@@ -0,0 +1,261 @@
+    if (!this.props.widget.content.result || this.props.isPreview) {
+      return;
+    }
+
+    d3
+      .select(chart.element)
+      .select('.c3-chart-arcs-title')
+      .attr('dy', -15)
+      .append('tspan')
+      .attr('dy', 30)
+      .attr('x', 0)
+      .attr('fill', '#666')
+      .text('SUM');
+
+    this.statusNode.addEventListener('mousemove', this.setCoords);
+  };
+
+  onStatusMouseOut = () => {
+    this.statusChart.revert();
+  };
+
+  onStatusMouseOver = (id) => {
+    this.statusChart.focus(id);
+  };
+
+  onStatusClick = (id) => {
+    this.statusChart.toggle(id);
+  };
+
+  getConfig = () => {
+    const { widget, container } = this.props;
+    const statusChartData = {};
+    const statusChartDataOrdered = [];
+    const statusChartColors = {};
+    const values = widget.content.result[0].values;
+
+    this.height = container.offsetHeight;
+    this.width = container.offsetWidth;
+    this.noAvailableData = false;
+
+    Object.keys(values).forEach((key) => {
+      if (key.indexOf('$executions$') > -1) {
+        const testStatus = key.split('$')[2].toUpperCase();
+        statusChartData[key] = +values[key];
+        statusChartColors[key] = COLORS[`COLOR_${testStatus}`];
+      }
+    });
+
+    statusChartData.statistics$executions$passed &&
+      statusChartDataOrdered.push([
+        'statistics$executions$passed',
+        statusChartData.statistics$executions$passed,
+      ]);
+    statusChartData.statistics$executions$failed &&
+      statusChartDataOrdered.push([
+        'statistics$executions$failed',
+        statusChartData.statistics$executions$failed,
+      ]);
+    statusChartData.statistics$executions$skipped &&
+      statusChartDataOrdered.push([
+        'statistics$executions$skipped',
+        statusChartData.statistics$executions$skipped,
+      ]);
+
+    if (
+      +statusChartDataOrdered.statistics$executions$total === 0 ||
+      !statusChartDataOrdered.length
+    ) {
+      this.noAvailableData = true;
+      return;
+    }
+
+    this.leftChartHidden = !statusChartDataOrdered.length;
+    this.statusItems = getDefectItems(statusChartDataOrdered);
+    this.statusConfig = {
+      data: {
+        columns: statusChartDataOrdered,
+        type: 'donut',
+        order: null,
+        colors: statusChartColors,
+      },
+      interaction: {
+        enabled: !self.isPreview,
+      },
+      padding: {
+        top: self.isPreview ? 0 : 85,
+      },
+      legend: {
+        show: false, // we use custom legend
+        position: 'bottom',
+      },
+      donut: {
+        title: 0,
+        label: {
+          show: false,
+          threshold: 0.05,
+        },
+      },
+      tooltip: {
+        grouped: false,
+        position: this.getStatusPosition,
+        contents: this.renderStatusContents,
+      },
+      onrendered: () => {
+        let total = 0;
+        if (this.statusChart) {
+          this.statusChart.data.shown().forEach((dataItem) => {
```

Comment:
```
let total = 0;
this.statusChart.data.shown().forEach((dataItem) => {
  total += dataItem.values[0].value;
});
```
yout can replace with
`const total = this.statusChart.data.shown().reduce((acc, dataItem) => acc + dataItem.values[0].value, 0);`